### PR TITLE
Update blob detection algorithm and refactor into separate class

### PR DESCRIPTION
**Pull request by Scoobydoo181**
Update blob detection algorithm to use several image preprocessing steps, as well as refactoring `detectEyes()` into a separate class with a constructor to initialize the various detectors as member variables. I also added simple mouse movement smoothing.

### CODE REVIEW

File: IrisSoftware/detectEyes.py [hunk 1]
```diff
@@ -182,8 +185,23 @@ def takePicture():
     cv2.imwrite("sampleFace.jpg", img)
     camera.release()
 
+def testBlobDetection2():
```
Comment (zacowan) [inline IrisSoftware/detectEyes.py]: Would you be willing to include a brief description of what each test case is doing in the function docstring?

**Review by zacowan — COMMENTED** [review]
Code looks good, though hard to tell exactly what this new test case is doing. I think adding an explanation of what each test case is doing could help improve clarity 😄

File: IrisSoftware/detectEyes.py (hunk 1 above)
Comment (Scoobydoo181) [reply IrisSoftware/detectEyes.py]: Done

**Review by zacowan — APPROVED** [review]
Looks great! Really clean refactor - thanks for adding the comments as well 🥇

**Pull request merged**